src/pydata_sphinx_theme/assets/styles/content/_tables.scss

```diff
@@ -6,9 +6,6 @@
 table {
   @include table-colors;
 
-  display: table;
-  overflow: auto;
-
```

Comment:
A bit of flip-flop history here:

- In #717, `display: block` and `overflow: auto` rules are applied to all table elements, which previously were restricted to tables within section elements
- In #818, `display: block` was changed to `display: table`

As I discuss in issue 1824, it's probably a [bad idea to change the display property of a table](https://github.com/pydata/pydata-sphinx-theme/issues/1824#issuecomment-2122587682) element since that might strip it of its semantics. And there's no need to explicitly put `display: table` on the table element, because that's what browsers assume.

As for the overflow rule, it only made sense when the table had `display: block`, as I discuss in PR #1827.

So to reduce head-scratching for future devs, I'm removing both rules.